File: __tests__/fixtures/output/clojure/clj_http/application-json.clj
(require '[clj-http.client :as client])

(client/post "https://httpbin.org/anything" {:content-type :json
                                             :form-params {:number 1
                                                           :string "f\"oo"
                                                           :arr [1 2 3]
                                                           :nested {:a "b"}
                                                           :arr_mix [1 "a" {:arr_mix_nested {}}]
                                                           :boolean false}})

File: src/targets/php/guzzle/fixtures/application-json.php
<?php
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'body' => '{"number":1,"string":"f\\"oo","arr":[1,2,3],"nested":{"a":"b"},"arr_mix":[1,"a",{"arr_mix_nested":{}}],"boolean":false}',
  'headers' => [
    'content-type' => 'application/json',
  ],
]);

echo $response->getBody();